### DISCUSSION TITLE
When to expect the latest Rspamd version for Ubuntu 24.04 Noble

### DISCUSSION THREAD

**Discussion by keyhelp** [rspamd/rspamd#5038] [Q&A] [answered]
Hello,

Is there any information on when to expect the release of the latest Rspamd versions for Ubuntu 24.04 Noble. Perhaps any of you know how long it has usually taken in the past for a new OS version to be supported.

I regularly check https://rspamd.com/apt-stable/dists/ but so far without success.

Many thanks in advance.

(I don't want to use the outdated version 3.8.1 that is included in the official Ubuntu repos).

**Comment by fatalbanana** [comment, marked as answer]
Builds for Noble are available from today with the release of 3.9.0
